api/v1alpha1/backstage_types.go

```diff
@@ -19,15 +19,15 @@ import (
 )
 
 const (
-	RuntimeConditionRunning string = "RuntimeRunning"
-	RuntimeConditionSynced  string = "RuntimeSyncedWithConfig"
-	RouteSynced             string = "RouteSynced"
-	LocalDbSynced           string = "LocalDbSynced"
-	SyncOK                  string = "SyncOK"
-	SyncFailed              string = "SyncFailed"
-	Deleted                 string = "Deleted"
-	EnvPostGresImage        string = "RELATED_IMAGE_postgresql"
-	EnvBackstageImage       string = "RELATED_IMAGE_backstage"
+	//RuntimeConditionRunning string = "RuntimeRunning"
```

Comment:
IMO RuntimeConditionRunning is exactly what you are doing with XSynced conditions. It is about "Running" (Deployed|Working), not about "Synced" :)

Reply:
As we discussed in #71 , the runtime object status will be deferred to a later release.

Reply:
> As we discussed in #71 , the runtime object status will be deferred to a later release.

Well, what I can see in this regard is your recent comment proposing to defer it, which does not look like "we discussed" :) .
Let's discuss, sure!

Reply:
@gazarenkov Thanks!